build-system/compile/single-pass.js

```diff
@@ -142,17 +143,20 @@ exports.getFlags = function(config) {
   return exports.getGraph(config.modules, config).then(function(g) {
     return flagsArray.concat(
         exports.getBundleFlags(g, flagsArray));
+  }).then(flagsArray => {
+    fs.writeFileSync('flags-array.txt', JSON.stringify(flagsArray, null, 2));
```

Comment:
I wonder if we should just remove this. Seems to be basically left over debugging code.

Reply:
Done.